### PR TITLE
routes page without query params redirects to root

### PR DESCRIPTION
**Pull request by Divinare**
Resolves: #210

### CODE REVIEW

File: src/components/sidebar/Sidebar.tsx [hunk 1]
```diff
@@ -35,6 +36,12 @@ interface ILinelistState {
 @inject('sidebarStore', 'routeStore', 'searchStore')
 @observer
 class Sidebar extends React.Component<ISidebarProps, ILinelistState> {
+
+    private renderRoutesView = () => {
+        const search = this.props.history.location.search;
+        return search === '' ?  <Redirect to='/' /> : <RoutesView />;
+    }
```
Comment (kronken) [inline src/components/sidebar/Sidebar.tsx]: The history things could be skipped if we utilize the navigator for this:
```
private renderRoutesView = () => {
    return !navigator.getQueryParam(QueryParams.routes)
        ? <Redirect to='/' /> : <RoutesView />;
}
```

Comment (kronken) [reply src/components/sidebar/Sidebar.tsx]: I would had put this logic inside the route view, and there just tell the navigator to navigate to '/'.

**Comment by Divinare** [comment]
Added todo to rename enum QueryParam. It could be done here: https://github.com/HSLdevcom/jore-map-ui/issues/137

Review comment fixed.

File: src/routing/queryParams.ts [hunk 2]
```diff
@@ -1,3 +1,4 @@
+// TODO: rename QueryParams https://en.wikipedia.org/wiki/Query_string
```
Comment (kronken) [inline src/routing/queryParams.ts]: Query string is the whole part of the URL string containing Query Parameters. A single label value pair is called a query parameter, so I think the current naming is OK.

**Pull request merged**